docs/configuration.md

```diff
@@ -23,6 +23,7 @@ Some of the configuration options provided in the file are stated below. These a
 
 - **logging:** An option to enable or disable logs.
 - **debug:** An option to enable or disable debug mode.
+- **threads:** The amount of threads that the app will use to run (the value should be greater than 0).
```

Comment:
Should we document to users that if the `threads` option is set to 0, we will automatically set it to `thread::available_parallelism()/2`?

Reply:
Well, it does say that the value should be greater than 0, but we can add a note about it.

Reply:
> Should we document to users that if the `threads` option is set to 0, we will automatically set it to `thread::available_parallelism()/2`?

I think we might not need to document that if we put it 0 part because it is just an error handling thing which is used to handle the 0 case value. Like if somebody does set it to 0 by mistake then it will handle the error by falling back to 0. :slightly_smiling_face: So I don't think we need to document that part. What do you say ?

Reply:
> > Should we document to users that if the `threads` option is set to 0, we will automatically set it to `thread::available_parallelism()/2`?
> 
> I think we might not need to document that if we put it 0 part because it is just an error handling thing which is used to handle the 0 case value. Like if somebody does set it to 0 by mistake then it will handle the error by falling back to 0. 🙂 So I don't think we need to document that part. What do you say ?

I agree with you.